server/ServerFSM.cpp

```diff
@@ -1188,7 +1194,42 @@ sc::result MPLobby::react(const LobbyUpdate& msg) {
             // limit count of AI
             auto max_ai = GetOptionsDB().Get<int>("network.server.ai.max");
             if (ai_count <= max_ai || max_ai < 0) {
-                m_lobby_data->m_players    = incoming_lobby_data.m_players;
+                if (sender->HasAuthRole(Networking::ROLE_HOST)) {
+                    // don't check host
```

Comment:
meaning the host can set any player to any empire, regardless of it nominally requiring an authenticated matching player name?

Reply:
Yes, I suppose Host is a `root` of the server.

Reply:
please clarify this in the comment

Reply:
Fixed.